packages/remix/src/ssr/authenticateRequest.ts

```diff
@@ -6,13 +6,19 @@ import { getEnvVariable } from '../utils';
 import type { LoaderFunctionArgs, RootAuthLoaderOptions, RootAuthLoaderOptionsWithExperimental } from './types';
 import { parseCookies } from './utils';
 
+const handleIsSatelliteBooleanOrFn = (opts: RootAuthLoaderOptionsWithExperimental, url: URL) => {
+  if (typeof opts.isSatellite === 'function') {
+    return opts.isSatellite(url);
```

Comment:
@panteliselef why do we need to pass all options? Judging from the name just passing `isSatellite` will do.